src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -83,23 +83,26 @@
  */
 public class EntityCitizen extends EntityAgeable implements INpc
 {
-    private static final float CONST_BED_HEIGHT = 0.6875f;
-    private static final float CONST_HALF_BLOCK = 0.5f;
+    private static final float CONST_BED_HEIGHT             = 0.6875f;
+    private static final float CONST_HALF_BLOCK             = 0.5f;
     private static final float CONST_SLEEPING_RENDER_OFFSET = -1.5f;
 
-    private static final int NINETY_DEGREE = 90;
-    private static final int HALF_ROTATION = 180;
+    private static final int NINETY_DEGREE  = 90;
+    private static final int HALF_ROTATION  = 180;
     private static final int THREE_QUARTERS = 270;
 
-    private static final DataParameter<Integer> DATA_TEXTURE         = EntityDataManager.<Integer>createKey(EntityCitizen.class, DataSerializers.VARINT);
-    private static final DataParameter<Integer> DATA_LEVEL           = EntityDataManager.<Integer>createKey(EntityCitizen.class, DataSerializers.VARINT);
-    private static final DataParameter<Integer> DATA_IS_FEMALE       = EntityDataManager.<Integer>createKey(EntityCitizen.class, DataSerializers.VARINT);
-    private static final DataParameter<Integer> DATA_COLONY_ID       = EntityDataManager.<Integer>createKey(EntityCitizen.class, DataSerializers.VARINT);
-    private static final DataParameter<Integer> DATA_CITIZEN_ID      = EntityDataManager.<Integer>createKey(EntityCitizen.class, DataSerializers.VARINT);
-    private static final DataParameter<String>  DATA_MODEL           = EntityDataManager.<String>createKey(EntityCitizen.class, DataSerializers.STRING);
-    private static final DataParameter<String>  DATA_RENDER_METADATA = EntityDataManager.<String>createKey(EntityCitizen.class, DataSerializers.STRING);
-    private static final DataParameter<Boolean> DATA_IS_ASLEEP = EntityDataManager.createKey(EntityCitizen.class, DataSerializers.BOOLEAN);
-    private static final DataParameter<BlockPos> DATA_BED_POS = EntityDataManager.createKey(EntityCitizen.class, DataSerializers.BLOCK_POS);
+    private static final int SPAWN_SEARCH_SIZE = 2;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "SPAWN_SEARCH_SIZE" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1068)